deploy/charts/crate-operator/README.md

```diff
@@ -21,10 +21,18 @@ Once Helm is properly set up, install the chart.
 
 ### Install from local folder
 
-```shell
+```
 helm install crate-operator crate-operator
 ```
 
+### Install from repo
+
+```
+helm repo add crate-operator https://crate.github.io/crate-operator
+helm search repo crate-operator
+helm install crate-operator/crate-operator
```

Comment:
I believe you also have to specify the name of the distribution you are installing? At least this simple install does not work for me:
```
$ helm install crate-operator/crate-operator
Error: INSTALLATION FAILED: must either provide a name or specify --generate-name

$ helm install my-crate-operator crate-operator/crate-operator --create-namespace
NAME: my-crate-operator
LAST DEPLOYED: Thu Sep 22 07:39:20 2022
NAMESPACE: crate-operator
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
Thank you for installing crate-operator.


```

Reply:
FWIW
```
$ helm version
version.BuildInfo{Version:"v3.7.0", GitCommit:"eeac83883cb4014fe60267ec6373570374ce770b", GitTreeState:"clean", GoVersion:"go1.16.8"}

```

Reply:
you're right, it should be `helm install crate-operator crate-operator/crate-operator`